configapp/configapp.py

```diff
@@ -21,16 +23,60 @@ def index():
 
 @app.route('/live')
 def live():
-    return render_template('live.html')
+    data = {}
+    subdirs = [d for d in os.listdir(predict_dir)
+               if os.path.isdir(os.path.join(predict_dir, d))]
+
+    if not subdirs:
+        return render_template('live.html', nopredictions=True)
+
+    subdirs.sort()
```

Comment:
@Adman It seems like the bigger question is whether that is safe to assume. If it is, please ignore this comment thread. if not, it should probably be dealt with in some other way.